### DISCUSSION TITLE
The guide for compiling with this library for Windows - on Github Actions

### DISCUSSION THREAD

**Discussion by omerlh** [adrg/go-wkhtmltopdf#20] [Show and tell]
Hey there! 
First, thanks for this library - it worked seamlessly for me! 
I used this library and was able to write a Github action that produce Windows binaries. It wasn't easy, so I want to share it here for the next person who might need it.
I wrote a [blog post](https://www.omerlh.info/2022/02/25/how-to-build-go-c-code-for-windows-using-github-actions/?_thumbnail_id=739), but the TL;DR is just here in the [workflow](https://github.com/omerlh/wizo-scheduler/blob/main/.github/workflows/go.yml) file.

Hope this will be helpful to others!

**Comment by adrg** [comment]
Hi @omerlh. Thank you for the contribution. This seems really useful for people who want to provide Windows binaries of their applications with their GitHub releases. I also enjoyed reading your blog post.
